.gitlab-ci.yml

```diff
@@ -32,6 +33,18 @@ ci-image:
     - docker build --tag $CI_IMAGE_DOCKER -f Dockerfile.gitlab .
     - docker push $CI_IMAGE_DOCKER
 
+# SECURITY
+
+create_key:
+  when: manual
```

Comment:
I'm not familiar with gitlab CI syntax, but I guess this will be a manual task which we are going to run only once in the lifetime of the project to generate GPG key?

Reply:
We'll generate it every time the GPG key expire or needs to be replaced. By default the GPG key will have a lifetime of 2 years IIRC.